core/modules/tests/testApp.py

```diff
@@ -28,9 +28,11 @@
 
 
 class AppTest(unittest.TestCase):
+
```

Comment:
No space needed between `class` and its docstring: https://developer.lsst.io/docs/py_docs.html#docstrings-of-classes-should-be-followed-but-not-preceded-by-a-blank-line
There are few other cases like this.